policyengine_us/variables/gov/states/mi/tax/income/exemptions/mi_exemptions.py

```diff
@@ -0,0 +1,54 @@
+from policyengine_us.model_api import *
+
+
+class mi_exemptions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Michigan Exemptions"
+    defined_for = StateCode.MI
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.legislature.mi.gov/Publications/TaxpayerGuide.pdf"
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.mi.tax.income.exemptions
+
+        # Personal Exemptions
+        person = tax_unit("tax_unit_size", period)
```

Comment:
Are you sure? I think it might just be for head and spouse. It doesn't appear that dependents are double-counted.